frontend/src/App.tsx

```diff
@@ -16,23 +17,58 @@ import {tablet} from "./Global";
 
 const App = (): JSX.Element => {
     const location = useLocation();
-
+    const [symbol, setSymbol] = React.useState("")
     const getSymbol = () => {
-        if (location.pathname === "/") return "/img/symbolY.svg"
-
         const symbols = [
             "/img/symbolA.svg",
+            "/img/symbolB.svg",
+            "/img/symbolC.svg",
+            "/img/symbolD.svg",
+            "/img/symbolE.svg",
+            "/img/symbolF.svg",
+            "/img/symbolG.svg",
+            "/img/symbolH.svg",
+            "/img/symbolI.svg",
+            "/img/symbolJ.svg",
+            "/img/symbolK.svg",
+            "/img/symbolL.svg",
+            "/img/symbolM.svg",
+            "/img/symbolN.svg",
+            "/img/symbolO.svg",
+            "/img/symbolP.svg",
+            "/img/symbolR.svg",
             "/img/symbolS.svg",
-            "/img/symbolY.svg"
+            "/img/symbolT.svg",
+            "/img/symbolU.svg",
+            "/img/symbolV.svg",
+            "/img/symbolW.svg",
+            "/img/symbolX.svg",
+            "/img/symbolY.svg",
+            "/img/symbolZ.svg",
         ]
         return symbols[Math.floor(Math.random() * symbols.length)]
     }
 
+    React.useEffect(() => {
+        setSymbol(getSymbol())
+    }, [location.pathname])
```

Comment:
Při změně se re-rendruje celá App, možná by to šlo dát do vlastní komponenty (do `Symbol`)?

Reply:
Nestačil by vlastní hook? Dám do vlastní komponenty, ale jsem zvědavý